modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -50,6 +56,7 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
   names(var.names) <- NULL
   multi.site.flag <- PEcAn.settings::is.MultiSettings(settings)
   readsFF<-NULL # this keeps the forward forecast
+  is.local <-PEcAn.remote::is.localhost(settings$host)
```

Comment:
Maybe I'm missing something in the design, but I was really surprised that this flag wasn't used more in this function. Does everything here really run completely fine on Remote without any need for database communication? I'm guessing this isn't the primary function being called but that this is called by something else that does all that run set up and output archiving?

Reply:
I suppose all input data would be processed and provided (since it should be querying formats from BETY) to SDA before transferring to remote?